web/src/lib/components/memory-page/memory-viewer.svelte

```diff
@@ -35,6 +35,7 @@
 	$: nextAsset = currentMemory?.assets[assetIndex + 1];
 
 	$: canAdvance = !!(nextMemory || nextAsset);
+	$: canGoBack = !!(previousMemory || previousAsset);
```

Comment:
It would be nice to keep the names consistent. I would suggest either this should be `canUnadvance` or `canAdvance` should be changed to `canMoveForwards`.

Reply:
Changed.